argobots.spec

```
@@ -96,6 +101,11 @@ rm -f %{buildroot}%{_libdir}/*.la
 %{_includedir}/*
 
 %changelog
+* Wed Jan 20 2021 B.Faccini <bruno.faccini@intel.com> - 1.1a1
```

Should include the `Release:` value and githash here too:
```suggestion
* Wed Jan 20 2021 B.Faccini <bruno.faccini@intel.com> - 1.1a1-1.g7fd1987
```